test/fixtures/box/labelMultiline.js

```diff
@@ -0,0 +1,105 @@
+module.exports = {
+  tolerance: 0.0070,
+  config: {
+    type: 'bar',
+    options: {
+      scales: {
+        x: {
+          labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July'],
+        },
+        y: {
+          display: false,
+          min: 0,
+          max: 25
+        }
+      },
+      plugins: {
+        annotation: {
+          annotations: {
+            box1: {
+              type: 'box',
+              xScaleID: 'x',
+              yScaleID: 'y',
+              xMin: 1.5,
+              xMax: 3.5,
+              yMin: 5,
+              yMax: 10,
+              backgroundColor: 'rgba(178, 255, 102, 0.5)',
+              borderColor: 'rgb(178, 255, 102)',
+              borderDash: [6, 6],
+              borderWidth: 5,
+              label: {
+                enabled: true,
+                content: ['This is a label', 'but this is multiline'],
+              }
+            },
+            box2: {
+              type: 'box',
+              xScaleID: 'x',
+              yScaleID: 'y',
+              xMin: 'May',
+              xMax: 'July',
+              yMin: 11,
+              yMax: 15,
+              backgroundColor: 'rgba(178, 255, 102, 0.5)',
+              borderColor: 'rgb(178, 255, 102)',
+              borderDash: [6, 6],
+              borderWidth: 5,
+              label: {
+                enabled: true,
+                content: ['This is a label', 'but this is multiline'],
+                align: 'left',
+                position: 'top'
+              }
+            },
+            box3: {
+              type: 'box',
+              xScaleID: 'x',
+              yScaleID: 'y',
+              xMin: -0.5,
+              xMax: 'May',
+              yMin: 16,
+              yMax: 20,
+              backgroundColor: 'rgba(178, 255, 102, 0.5)',
+              borderColor: 'rgb(178, 255, 102)',
+              borderDash: [6, 6],
+              borderWidth: 5,
+              label: {
+                enabled: true,
+                content: ['This is a label', 'but this is multiline'],
+                font: {
+                  size: 18
```

Comment:
The font used by `spriteText` is 16 pixels and is not scaled, so maybe using that as size would work better?

Reply:
Doesn't change too mucch therefore I think it's better to remove font options.